internal/envoy/v3/stats.go

```diff
@@ -80,3 +146,20 @@ func StatsListener(address string, port int) *envoy_listener_v3.Listener {
 		SocketOptions: TCPKeepaliveSocketOptions(),
 	}
 }
+
+func serviceStatsRoute(prefix string) *envoy_route_v3.Route {
```

Comment:
yup I agree. That's what it got called initially. Not sure if we want to split off this admin stuff into a new one? But that seems like a waste. Also not sure if folks have the `service-stats` in any dashboards, etc.

Reply:
I think to be fair we need to do a proper deprecation of this field. I really doubt this would be a breaking change for most, but not 100% sure. Ok to make a followup PR?

Reply:
//cc @sunjayBhatia

Reply:
Yep 👍

Reply:
sounds good to me, should err on the side of being careful 👍🏽